sipa/templates/base.html

```diff
@@ -357,6 +357,13 @@ <h2>{{ _("Kontakt") }} </h2>
                                 <hr>
 
                                 <div class="row">
+                                    <div class="col-xs-4 text-center">
```

Comment:
Needs to be changed to `col-xs-3`. This also applied for the other 3 icons.

Reply:
Done.